src/blitter/base.hpp

```diff
@@ -170,7 +170,7 @@ class Blitter : public SpriteEncoder {
 	 * @param height The height of the buffer-to-be.
 	 * @return The size needed for the buffer.
 	 */
-	virtual int BufferSize(int width, int height) = 0;
+	virtual size_t BufferSize(int width, int height) = 0;
```

Comment:
Arguably the arguments should also be unsigned, a negative buffer dimension doesn't make sense.

Reply:
Totally agree, but I considered it out-of-scope of this PR :) Otherwise it becomes such a rabbit-hole :P

In good news, most of the time width/height are unsigned .. just sometimes they are not ;)